processing/src/main/java/io/druid/segment/NullHandlingHelper.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.segment;
+
+import com.google.common.base.Strings;
+import com.google.inject.Inject;
+import io.druid.query.aggregation.Aggregator;
+import io.druid.query.aggregation.BufferAggregator;
+import io.druid.query.aggregation.DoubleAggregateCombiner;
+import io.druid.query.aggregation.LongAggregateCombiner;
+import io.druid.query.aggregation.NullFilteringAggregator;
+import io.druid.query.aggregation.NullFilteringBufferAggregator;
+import io.druid.query.aggregation.NullableAggregator;
+import io.druid.query.aggregation.NullableBufferAggregator;
+import io.druid.query.aggregation.NullableDoubleAggregateCombiner;
+import io.druid.query.aggregation.NullableLongAggregateCombiner;
+import io.druid.query.aggregation.NullableObjectAggregateCombiner;
+import io.druid.query.aggregation.ObjectAggregateCombiner;
+
+public class NullHandlingHelper
+{
+
+  // use these values to ensure that convertObjectToLong(), convertObjectToDouble() and convertObjectToFloat()
+  // return the same boxed object when returning a constant zero.
+  public static final Double ZERO_DOUBLE = 0.0d;
```

Comment:
this seems to be duplicated, can we use this one or `io.druid.segment.DimensionHandlerUtils#ZERO_DOUBLE`

Reply:
this still open -> https://github.com/druid-io/druid/blob/7552a611f18fe460136f1b4953ef69d64d9e8718/processing/src/main/java/io/druid/segment/DimensionHandlerUtils.java#L48